lib/util/lib/slab.c

```diff
@@ -1096,8 +1094,7 @@ kmem_cache_fini(void)
 
     kmc.kmc_pagecache = NULL;
 
-    kmc_zone_foreach(zone, next, &kmc.kmc_zones)
-        kmem_cache_destroy((void *)zone);
+    kmc_zone_foreach(zone, next, &kmc.kmc_zones) kmem_cache_destroy((void *)zone);
```

Comment:
I guess you need to add this to your "for-each" list?

Reply:
Will do.